.github/workflows/qemu-kvm-integration-tests.yml

```
@@ -109,6 +139,24 @@ jobs:
               echo "$f"
           done < batch.report
 
+      - name: Run container tox integration tests
+        if: steps.check_platform.outputs.supported && startsWith(matrix.scenario.env, 'container')
+        run: |
+          set -euo pipefail
+          # HACK: debug.py/profile.py setup is broken
+          export LSR_CONTAINER_PROFILE=false
+          export LSR_CONTAINER_PRETTY=false
+          rc=0
+          for t in tests/tests_*.yml; do
+              if tox -e ${{ matrix.scenario.env }} -- --image-name ${{ matrix.scenario.image }} $t > ${t}.log 2>&1; then
+                  echo "$(basename $t): pass"
```

```suggestion
                  echo "PASS: $(basename $t)"
```
should be consistent with the way qemu test results are reported - https://github.com/linux-system-roles/sudo/blob/888087f0cfdc8483b87698252685c81b919c647f/.github/workflows/qemu-kvm-integration-tests.yml#L134